internal/eventstore/spooler/spooler_test.go

```diff
@@ -0,0 +1,335 @@
+package spooler
+
+import (
+	"context"
+	"fmt"
+	"github.com/caos/zitadel/internal/eventstore"
+	"github.com/caos/zitadel/internal/eventstore/models"
+	"github.com/caos/zitadel/internal/eventstore/spooler/mock"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+)
+
+type testHandler struct {
+	cycleDuration time.Duration
+	processSleep  time.Duration
+	processError  error
+	queryError    error
+	viewModel     string
+}
+
+func (h *testHandler) ViewModel() string {
+	return h.viewModel
+}
+func (h *testHandler) EventQuery() (*models.SearchQuery, error) {
+	return nil, h.queryError
+}
+func (h *testHandler) Process(*models.Event) error {
+	<-time.After(h.processSleep)
+	return h.processError
+}
+func (h *testHandler) MinimumCycleDuration() time.Duration { return h.cycleDuration }
+
+type eventstoreStub struct {
+	events []*models.Event
+	err    error
+}
+
+// Healthz returns status OK as soon as the service started
+func (es *eventstoreStub) Health(ctx context.Context) error {
+	return nil
+}
+
+// Ready returns status OK as soon as all dependent services are available
```

Comment:
wrong comment

Reply:
i remove the comments, because its just for tests and i dont really do something in this funcs